packages/hoprd/src/api/v2/utils.ts

```diff
@@ -4,24 +4,24 @@
  * reuse the code for commands, will change if said
  * otherwise.
  */
-export const STATUS_CODES = {
+export enum STATUS_CODES {
```

Comment:
why was this necessary?

Reply:
Enums provide better typings and make exhaustive type checks possible. Fixing issues resulting from incorrect conversions is the main motivation for this PR